src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -153,10 +153,16 @@ public Response describeManageDataAccessRequests(@Auth AuthUser authUser) {
     @GET
     @Produces("application/json")
     @Path("/manage/v2")
-    @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER})
-    public Response describeManageDataAccessRequestsV2(@Auth AuthUser authUser) {
-        List<DataAccessRequestManage> dars = dataAccessRequestService.describeDataAccessRequestManageV2(authUser);
-        return Response.ok().entity(dars).build();
+    @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER, SIGNINGOFFICIAL})
+    public Response describeManageDataAccessRequestsV2(@Auth AuthUser authUser, @QueryParam("roleName") Optional<String> roleName) {
+        try {
+            User user = userService.findUserByEmail(authUser.getName());
+            String roleNameValue = roleName.isPresent() ? roleName.get() : null;
```

Comment:
Intellij should be recommending this for you:
```suggestion
            String roleNameValue = roleName.orElse(null);
```